kurtosis/src/nodes/consensus/beacond/scripts/multiple-premined-deposits.sh

```diff
@@ -21,15 +21,15 @@
 
 
 /usr/bin/beacond init --chain-id $BEACOND_CHAIN_ID $BEACOND_MONIKER --home /tmp/config0/.beacond --consensus-key-algo $BEACOND_CONSENSUS_KEY_ALGO
-/usr/bin/beacond genesis add-premined-deposit --home /tmp/config0/.beacond
+/usr/bin/beacond genesis add-premined-deposit $DEPOSIT_AMOUNT $WITHDRAWAL_ADDRESS --home /tmp/config0/.beacond
```

Comment:
@calbera should we cross check with ops that this change is fine on their side? Just to avoid breaking tooling they may have around

Reply:
yes exactly, hence @chuck-bear is informed. This will break tooling but is required.